.github/workflows/e2e_tests.yaml

```diff
@@ -123,6 +123,12 @@ jobs:
             yarn test:android:run
         env:
           API_LEVEL: ${{ matrix.api-level }}
+
+      - name: Test coverage
```

Comment:
If we want to add this, then we need this for Android and IOS. Not only for Android.

Reply:
Will do both, thanks for the feedback